federator/README.md

```diff
@@ -58,6 +58,29 @@ You need to change `"<YOUR NODE HOST AND RPC PORT>"` for the url of your node fo
 `Remember to do it for both networks`.
 Also you need to create a `federators.key` file with the federator private in it.
 
+### Development
+- In your development environment you must have 2 blockchains running (ganache is ok)
+- To start, go to the `bridge` directory and run
+```shell
+$ cd bridge
```

Comment:
I'd remove this line, since you said `go to the bridge directory` before

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/253/commits/27de33d8b9b3cc4d2f3be370aa7e25ebe4eb1c4b